src/WebJobs.Script.WebHost/Scale/TableEntityConverter.cs

```diff
@@ -91,9 +91,12 @@ public static bool TryGetEntityProperty(JProperty property, out EntityProperty e
                     entityProperty = new EntityProperty(value.ToObject<Guid>());
                     return true;
                 case JTokenType.Integer:
-                    entityProperty = new EntityProperty(value.ToObject<int>());
+                    // to handle both ints and longs, we normalize integer values
+                    // to type long
+                    entityProperty = new EntityProperty(value.ToObject<long>());
```

Comment:
Is this related to the current issue?

Reply:
No - please see the PR description above.